discovery_engine_api_tests/tests/test_front_office.py

```diff
@@ -0,0 +1,44 @@
+import allure
+import pytest
+from handlers.api_handler import ApiHandler
+from model.interactions import Interactions
+from model import documents
+from utils import test_utils as su
+from utils import assert_utils as au
+
+
+@allure.suite("Front Office Test Suite")
+class TestFrontOffice:
+    api_handler = ApiHandler()
+
+    @pytest.fixture
+    def generate_post_docs(self):
+        doc_dict = documents.generate_docs(1).popitem()
+        request = self.api_handler.post_document(doc_dict[1])
+        au.assert_status_code_equals(request.status_code, 201)
+        return doc_dict[0]
+
+    @allure.severity(allure.severity_level.CRITICAL)
+    def test_positive_interaction(self, generate_post_docs):
+        user_id = su.generate_random_alphanumerical(6)
+        positive_interaction = Interactions(generate_post_docs, "Positive").to_json()
+        request = self.api_handler.interact_with_documents(user_id, positive_interaction)
+        au.assert_status_code_equals(request.status_code, 204)
+
+    @allure.severity(allure.severity_level.NORMAL)
+    @pytest.mark.skip(reason="todo: create ticket")
+    def test_invalid_doc_id(self):
+        user_id = su.generate_random_alphanumerical(6)
+        interaction_with_random_id = Interactions(su.generate_random_alphanumerical(36), "Positive").to_json()
```

Comment:
:wrench: this is not guaranteed to be an invalid document id (as in the id is invalid not the document doesn't exist).

And due fundamental race conditional around viewing, deletion  and ingestion interacting with a non existing document will not fail (through it will log a  warning).

Reply:
(and if we would fail on non existing documents it would not be a 400 error but some 40x error, likely 409)

Reply:
@rustonaut ok i'll mark this test as "needs to be updated" and leave it with @skip mark